### PR TITLE
Update projects page

### PR DESCRIPTION
**Pull request by malinatran**

### CODE REVIEW

**Review by davidvanbeveren — DISMISSED** [review]

**Review by davidvanbeveren — DISMISSED** [review]

File: src/pages/projects/index.tsx [hunk 1]
```diff
@@ -196,10 +196,10 @@ const Projects: PageWithLayout<ProjectsProps> = ({
     <>
       <NextSeo title="Projects" />
       <FirstSubSection header="Our projects">
-        We&apos;re constantly working on new projects every month, whether
-        they&apos;re ideas of our own or supporting organizations and activists
-        that reach out to us. Below is a list of our work, and which of our
-        teams worked on it.
+        We&apos;re constantly working on new projects and maintaining existing
+        ones, whether they're in-house or in support of other organizations.
```
Comment (Headscracher) [inline src/pages/projects/index.tsx]:
```suggestion
        ones, whether they&apos;re in-house or in support of other organizations.
```

**Review by Headscracher — CHANGES_REQUESTED** [review]

File: src/pages/projects/index.tsx (hunk 1 above)
Comment (davidvanbeveren) [reply src/pages/projects/index.tsx]: @Headscracher Same for this one? ’

File: src/components/layout/projects/layout.tsx [hunk 2]
```diff
@@ -57,10 +57,11 @@ const ProjectsLayout: Layout = ({ children }) => {
...
           <p className="my-5 text-xl">
             <b>We&apos;re all ears!</b> We&apos;d love to hear your ideas for
-            projects that can help empower the animal rights movement, if we
-            like it, we&apos;ll build it! While we do get a lot of project
-            suggestions every month, we do consider every single one. If you
-            rather build it yourself and want advice, no problem - get in touch!
+            projects that can help empower the animal protection movement. We
+            consider each project sent to us, and determine how we can help you
+            actualize your idea. Whether you are seeking a team to build from
+            beginning to end, or simply looking for advice to get started, get
```
Comment (malinatran) [inline src/components/layout/projects/layout.tsx]:
```suggestion
            start to finish, or looking for advice to get your project off the ground, get
```

File: src/pages/projects/index.tsx (hunk 1 above)
Comment (Headscracher) [reply src/pages/projects/index.tsx]: Yes same for this one.

**Pull request merged**